tests/test_schema.py

```diff
@@ -56,14 +59,18 @@ def _validateSchema(self, dataset, dataId, tableName):
         outputColumnNames = set(df.columns.to_list())
         self.assertEqual(outputColumnNames, set(expectedColumns.keys()), f"{info} failed")
 
-        # the data type mapping from felis datatype to pandas
+        # Pandas 2.1.0 changed the datetime dtype from [ns] -> [us],
+        # so we need a pandas version-aware test here.
+        pdVersion = packaging.version.parse(pd.__version__)
```

Comment:
I am continually told that `__version__` is unreliable and we should be using `importlib.metadata` to get package versions (which is what utils now does).

```python
>>> import importlib.metadata
>>> importlib.metadata.version("pandas")
'2.0.3'
```

Reply:
Well, pandas is also unreliable as regards to datatypes...

But yes, I'll add a third import to this test, just to get the pandas type to pass.